src/funfuzz/js/build_options.py

```diff
@@ -263,12 +256,9 @@ def areArgsValid(args):  # pylint: disable=invalid-name,missing-param-doc,missin
         return False, 'Making an optimized, non-optimized build would be contradictory.'
     if not args.enableDbg and args.disableOpt:
         return False, 'Making a non-debug, non-optimized build would be kind of silly.'
-    if sps.isARMv7l and not args.enable32:
-        return False, '64-bit ARM builds are not yet supported.'
```

Comment:
Minor nit here but why not raise a parser error when checking these values instead of returning a bool and error message?

Reply:
Interesting suggestion. I haven't thought of this before, I'll add a note considering this.